### ISSUE TITLE
Complement Printing

### ISSUE THREAD

**Issue by gxyd** [sympy/sympy#9877]
See [this](https://github.com/sympy/sympy/issues/7486#issuecomment-135709498) comment
```
>>> Union(Complement(Interval(1, 2), FiniteSet(x)), Interval(2, 3, True, True))
(2, 3) U [1, 2] \ {x}                 # this should be like ->     ((2, 3) U ([1, 2] \ {x}))
```

```
>>> a = Intersection(FiniteSet(x), FiniteSet(y), Complement(FiniteSet(z), Interval(1, 2)))
>>> a
{x} ∩ {y} ∩ {z} \ [1, 2]    # should return {x} ∩ {y} ∩ ({z} \ {1, 2})
```

**Comment by hargup** [comment]
@gxyd this is a low priority easy to fix issue, you should leave such issues for new people.

**Comment by gxyd** [comment]
@preetskhalsa97 you can take this up.

**Comment by gxyd** [comment]
@kshitij10496 you can take this.

**Comment by kshitij10496** [comment]
@gxyd Thanks.

Onto it.

**Comment by kshitij10496** [comment]
@hargup  @gxyd Sent a PR. 
Please review it.

**Comment by gxyd** [comment]
Closing, fixed by #10178
